### PR TITLE
Add Tesla example

### PR DESCRIPTION
**Pull request by lukicdarkoo**
This example demonstrates how to use AckermannDrive with ROS2 and Webots.

Ackerman messages are not released for Rolling. We can keep it as is in the master branch, but exclude the package when realizing to Rolling.

### CODE REVIEW

**Comment by omichel** [comment]
Rolling tests are failing.

**Comment by lukicdarkoo** [comment]
> Rolling tests are failing.

Yes, yes, I know, I put a note about that. We don't think we can avoid it. We have to publish different versions for Rolling and Foxy.

File: webots_ros2_tesla/worlds/tesla_world.wbt [hunk 1]
```diff
@@ -0,0 +1,1515 @@
+#VRML_SIM R2021b utf8
+WorldInfo {
+  info [
+    "Autonomous Vehicle Simulation"
+    "The simple controller example uses an on-board camera to follow the yellow road lines and a SICK sensor to avoid the obstacles."
+    "The control of the vehicle is done using the driver library."
+    "The vehicle based on the Car PROTO is modelled with realistic physics properties: motor torques, body mass, friction forces, suspensions, etc."
+  ]
+  title "City"
```
Comment (omichel) [inline webots_ros2_tesla/worlds/tesla_world.wbt:9]: I believe the title (and description) should be more specific to the Tesla Model 3 car.

File: webots_ros2_tesla/worlds/tesla_world.wbt [hunk 2]
```diff
@@ -0,0 +1,1515 @@
...
+      coulombFriction [
+        8
+      ]
+      softCFM 1e-05
+      bumpSound ""
+      rollSound ""
+      slideSound ""
+    }
+    ContactProperties {
+      material2 "RangeRoverSportSVRWheels"
```
Comment (omichel) [inline webots_ros2_tesla/worlds/tesla_world.wbt:54]: Do we really need all those contact properties?

**Review by omichel — CHANGES_REQUESTED** [review]
Thank you.

File: webots_ros2_tesla/worlds/tesla_world.wbt (hunk 2 above)
Comment (lukicdarkoo) [reply webots_ros2_tesla/worlds/tesla_world.wbt:54]: Good point

**Review by omichel — APPROVED** [review]
Thank you.

**Pull request merged**